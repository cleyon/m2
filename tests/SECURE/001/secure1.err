m2:secure1.m2:10:(xeq_cmd__shell) @shell - Security violation
