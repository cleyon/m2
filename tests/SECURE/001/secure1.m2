@define __SECURE__ 1
Secure level is @__SECURE__@
@if exists(/etc/passwd)
Password file exists
@else
Password file does not exist
@fi
@shell EOD
date
EOD
@define __SECURE__ 0
Secure level is @__SECURE__@
All done
