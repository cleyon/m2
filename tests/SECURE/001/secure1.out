Secure level is 1
Password file exists
Secure level is 1
All done
